eth-contracts/contracts/TokenLocker.sol

```diff
@@ -49,14 +54,17 @@ contract TokenLockerRaw {
         uint256 bridgeFee
     );
 
-    constructor(
+    function initialize(
         address ckbSpvAddress,
         uint64 numConfirmations,
         bytes32 recipientCellTypescriptCodeHash,
         uint8 typescriptHashType,
```

Comment:
Should rename `typescriptHashType` to `recipientCellTypescriptCodeHashType` and add a parameter `bridgeCellLockscriptHashType`